api/libproject/devstart.rb

```diff
@@ -1261,48 +1261,49 @@ def backfill_billing_project_owners(cmd_name, *args)
     :fn => ->(*args) {backfill_billing_project_owners("backfill-billing-project-owners", *args)}
 })
 
-def update_user_registered_status(cmd_name, args)
+def update_user_disabled_status(cmd_name, args)
   common = Common.new
   op = WbOptionsParser.new(cmd_name, args)
   op.add_option(
     "--project [project]",
     ->(opts, v) { opts.project = v},
-    "Project to update registered status for"
+    "Project to update disabled status for"
   )
   op.add_option(
     "--disabled [disabled]",
     ->(opts, v) { opts.disabled = v},
-    "Disabled state to set: true/false."
+    "Disabled state to set: true/false"
   )
   op.add_option(
-    "--account [account]",
+    "--account [admin_account_email]",
```

Comment:
Argh! This must have been the thing missing from the command that the script didn't tell us about. Didn't think to pull the script's help in the middle of the night.

Reply:
If it's any consolation, the script's help docs were pretty inaccurate. Also it was hitting the wrong URL so it would not have worked without code modifications